src/main/java/net/openhft/posix/internal/jnr/JNRPosixAPI.java

```diff
@@ -117,6 +117,7 @@ private int mlock2_(long addr, long length, boolean lockOnFault) {
 
     @Override
     public boolean mlock(long addr, long length) {
+        if(Jvm.isAzul()) return true; // no-op on Azul, ignore
```

Comment:
Suggest we Jvm.warn

Reply:
Agreed, but maybe only the first time it's called?

Reply:
added warns for mlock/2
mlock/2  are (should be) very infrequently called for any app, so will log on each call rather than just once (which could be missed)